java-extras/src/main/java/org/triplea/java/ViewModelListener.java

```diff
@@ -0,0 +1,54 @@
+package org.triplea.java;
+
+/**
```

Comment:
The comment and example on this interface are important. Please scrutinize whether it all makes sense.